prow/tide/tide.go

```diff
@@ -822,13 +822,27 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 	for i, pr := range prs {
 		log := log.WithFields(pr.logFields())
 		mergeMethod := c.config().Tide.MergeMethod(sp.org, sp.repo)
-		if squashLabel := c.config().Tide.SquashLabel; squashLabel != "" {
+		squashLabel := c.config().Tide.SquashLabel
+		rebaseLabel := c.config().Tide.RebaseLabel
+		mergeLabel := c.config().Tide.MergeLabel
+		if squashLabel != "" || rebaseLabel != "" || mergeLabel != "" {
+			labelCount := 0
 			for _, prlabel := range pr.Labels.Nodes {
-				if string(prlabel.Name) == squashLabel {
+				switch string(prlabel.Name) {
```

Comment:
Forgive my golang rustiness asking this one, but is a switch where multiple cases are the same legal? The conditional around the whole things ensures that at least ONE of `squashLabel`/`rebaseLabel`/`mergeLabel` is non-empty, but it's possible that 2/3 are empty (or that they're non-empty but otherwise identical)

Probably worth checking that if they exist they aren't identical since that would be another way to accidentally get some surprising behavior with a non-obvious cause.

Reply:
Yeah, this does seem confusing, and is making things a bit more complex than I had hoped here :( 

go seems to be ok with allowing duplicate cases for variables (the same variable or variables with the same value), though it would be illegal for something like multiple `case ""`'s

Reply:
Here we could use an empty switch, fall through, and a counter (`labelCount`)... if the counter is `> 1`, post a comment warning multiple label existence?

Fall through is not enabled by default in go: https://golang.org/test/switch.go